graders/python/requirements.txt

```diff
@@ -2,13 +2,13 @@ beautifulsoup4==4.12.3
 bokeh==3.6.0
 colormath==3.0.0
 defusedxml==0.7.1
-Faker==30.3.0
-folium==0.17.0
-ipython==8.28.0
+Faker==30.8.2
```

Comment:
amazingly not a major version bump

Reply:
From what I understand, Faker will create a new major version if, for the same seed, the values returned for any existing provider are expected to change. It seems in this case all changes were to new providers instead of updating existing providers.